### PR TITLE
Rename `frontend.server.js` to `server.js`

### PR DESCRIPTION
**Pull request by arelra**
## What does this change?

Rename the dist server output file from `frontend.server.js` to `server.js`

## Why?

'frontend' is a potentially confusing and unnecessary prefix for the server app.

I assume it's a hangover from ye olde Frontend.

### CODE REVIEW

**Review by OllysCoding — APPROVED** [review]
+1 This is a very sensible change :)

File: dotcom-rendering/makefile [hunk 1]
```diff
@@ -109,11 +109,11 @@ storybook-dev: clear clean-dist install
 
 cypress: clear clean-dist install build
 	$(call log, "starting frontend PROD server for Cypress")
```
Comment (georgeblahblah) [inline dotcom-rendering/makefile:111]:
```suggestion
	$(call log, "starting PROD server for Cypress")
```

**Review by georgeblahblah — APPROVED** [review]
Makes sense! I have suggested 2 changes to some log statements which aren't related to your changes, but we could change at the same time!

File: dotcom-rendering/makefile (hunk 1 above)
Comment (arelra) [reply dotcom-rendering/makefile:111]: Good spot thanks!

File: dotcom-rendering/makefile [hunk 2]
```diff
@@ -62,7 +62,7 @@ start: install
 start-prod:
 	$(call log, "starting PROD server...")
 	@echo '' # just a spacer
-	NODE_ENV=production /usr/local/node/pm2 start --uid dotcom-rendering --gid frontend dist/server.js
+	NODE_ENV=production /usr/local/node/pm2 start --uid dotcom-rendering --gid server dist/server.js
```
Comment (arelra) [inline dotcom-rendering/makefile:65]: I've renamed the pm2 `gid` here also. I'll assume nothing is expecting `frontend` (logs, analytics etc) but I'll connect to a CODE instance to sanity check.

Comment (arelra) [reply dotcom-rendering/makefile]: `gid` is the user's group not the pm2 id for the process as I had assumed. Changing this preventing the app from starting up so I've reverted to `--gid frontend`.

**Pull request merged**